internal/cmd/beta/network-area/routes/describe/describe.go

```diff
@@ -0,0 +1,147 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	routeIdArg = "ROUTE_ID"
+
+	organizationIdFlag = "organization-id"
+	networkAreaIdFlag  = "network-area-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	OrganizationId *string
+	NetworkAreaId  *string
+	RouteId        string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Shows details of a static route in a STACKIT Network Area (SNA)",
+		Long:  "Shows details of a static route in a STACKIT Network Area (SNA).",
+		Args:  args.SingleArg(routeIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Show details of a static route with ID "xxx" in a STACKIT Network Area with ID "yyy" in organization with ID "zzz"`,
+				`$ stackit beta network-area routes describe xxx --organization-id zzz --network-area-id yyy`,
```

Comment:
```suggestion
				`$ stackit beta network-area routes describe xxx --network-area-id yyy --organization-id zzz`,
```

Reply:
or we should change the order in explanation part. It will be nice to have flags in same order